src/CMakeLists.txt

```diff
@@ -196,12 +196,7 @@ find_package(CapnProto REQUIRED)
 include_directories(${CAPNP_INCLUDE_DIRS})
 # List all .capnp files here. The C++ files will be generated and included
 # when compiling later on.
-set(CAPNP_SPECS
-    nupic/algorithms/SpatialPoolerProto.capnp
-    nupic/math/SparseBinaryMatrixProto.capnp
-    nupic/math/SparseMatrixProto.capnp
-    nupic/utils/RandomProto.capnp
-)
+file(GLOB CAPNP_SPECS nupic/proto/*.capnp)
```

Comment:
Bear in mind `GLOB` is somewhat deprecated.  From http://www.cmake.org/cmake/help/v3.0/command/file.html?highlight=glob#file 

> We do not recommend using GLOB to collect a list of source files from your source tree. If no CMakeLists.txt file changes when a source is added or removed then the generated build system cannot know when to ask CMake to regenerate.